index.html

```diff
@@ -1424,7 +1425,8 @@ <h2 id="objects">Object Functions</h2>
       <p id="pairs">
         <b class="header">pairs</b><code>_.pairs(object)</code>
         <br />
-        Convert an object into a list of <tt>[key, value]</tt> pairs.
+        Convert an object into a list of <tt>[key, value]</tt> pairs. Inverse of first
+        usage of <code>_.object</code>. 
```

Comment:
Hi @captbaritone , thanks for the feedback. Is this clearer? I tried to match the wording of other "opposites" such as in _.unescape and _.unzip.

 Also -- I edited the corresponding comments in underscore.js so that the annotated source will stay consistent w/ the main page. Take a look!